Dockerfile

```diff
@@ -7,7 +7,7 @@ LABEL org.opencontainers.image.source https://github.com/tkhq/frames
 
 COPY nginx.conf /etc/nginx/nginx.conf
 
-COPY recovery /usr/share/nginx/recovery
+COPY auth /usr/share/nginx/auth
 COPY export /usr/share/nginx/export
```

Comment:
I think we need to keep supporting recovery.turnkey.com. So let's "cheat":
```
COPY auth /usr/share/nginx/recovery
```
:)